configs/textdet/fcenet/fcenet_r50dcnv2_fpn_1200e_ctw1500.py

```diff
@@ -0,0 +1,136 @@
+fourier_degree = 5
+model = dict(
+    type='FCENet',
+    pretrained='torchvision://resnet50',
+    backbone=dict(
+        type='ResNet',
+        depth=50,
+        num_stages=4,
+        out_indices=(1, 2, 3),
+        frozen_stages=-1,
+        norm_cfg=dict(type='BN', requires_grad=True),
+        norm_eval=True,
+        style='caffe',
+        dcn=dict(type='DCNv2', deform_groups=2, fallback_on_stride=False),
+        stage_with_dcn=(False, True, True, True)),
+    neck=dict(
+        type='FPN',
+        in_channels=[512, 1024, 2048],
+        out_channels=256,
+        add_extra_convs=True,
+        extra_convs_on_inputs=False,  # use P5
+        num_outs=3,
+        relu_before_extra_convs=True,
+        act_cfg=None),
+    bbox_head=dict(
+        type='FCEHead',
+        in_channels=256,
+        scales=(8, 16, 32),
+        loss=dict(type='FCELoss'),
+        fourier_degree=fourier_degree,
+        ))
+
+train_cfg = None
+test_cfg = None
+
+dataset_type = 'IcdarDataset'
+data_root = '/home/chen/mmocr_date_release/mmocr41/mmocr/data/ctw1500/'
```

Comment:
```suggestion
data_root = 'data/ctw1500'
```
do not use the absolute path which contains private information.
you can soft link you folder into data/